tox.ini

```diff
@@ -1,25 +1,25 @@
 [tox]
-envlist = py27-dj15, py27-dj16
+envlist = py{26,27}-django16, py27-django17, pep8
+skip_missing_interpreters=True
 
 [testenv]
 commands =
-    coverage erase
+    - coverage erase
     python runtests.py
-    coverage report
-deps =
+    - coverage report
+deps=
+    py26: unittest2
+    ; Something happens that sometimes premailer (dependence of emailit) does
+    ; not install ordereddict on tox envs with Python 2.6.x, so I add it here.
+    ; If we found that issue on normal environments we must handle it as a bug
+    ; and add ordereddict to setup.py
+    py26: ordereddict
+    django16: django<1.7,>=1.6
+    django17: django<1.8,>=1.7
+    ; TODO: after merge the PR in aldryn-categories remove this line
+    https://github.com/aldryn/aldryn-categories/archive/feature/fk_fields.zip
```

Comment:
should be reviewed , relates to https://github.com/aldryn/aldryn-categories/pull/9